Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:lts-alpine
+
+RUN apk add --no-cache --virtual build-dependencies bash git python make g++ ca-certificates
+
+COPY yarn.lock package.json ./
+RUN yarn install --frozen-lockfile && yarn cache clean
+
+COPY . .
+RUN yarn prepack
+
+ENTRYPOINT ["bash", "-c"]
+CMD ["npx", "truffle", "exec", "scripts/verify_streamed_orderbook.js"]
```

Comment:
No newline at end of file!?

Reply:
There is one... When there isn't there is a little symbol: 
![image](https://user-images.githubusercontent.com/4210206/81911688-f2a5ae00-95cd-11ea-8aff-bc7b4ca93869.png)

Reply:
Nice symbol!